python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -63,6 +63,9 @@
     pytest_xfail = pytest.mark.skipif
 
 
+rng = np.random.default_rng(seed=0)
+
+
```

Comment:
*nit*: Would prefer not to have a global rng in this test file.

Reply:
Dropped it.